bin/local-env/docker-compose.yml

```diff
@@ -28,7 +28,7 @@ services:
     command: tail -f /dev/null
 
   mysql:
-    image: mysql:5.7
+    image: mariadb:10
```

Comment:
The `mysql` Docker image doesn't have an arm64 compatible build yet, but `mariadb` does so I went with that.